src/Parser/ParserOptions.cs

```diff
@@ -16,6 +16,24 @@ public ParserOptions()
         /// Sets up the parser options to work with the given Visual Studio toolchain.
         public void SetupMSVC(VisualStudioVersion vsVersion = VisualStudioVersion.Latest)
```

Comment:
Now that we override the VS version in the switch below, we don't need a parameter. Turn it into a local.